### DISCUSSION TITLE
Is it possible to setup multiple host per tenant?

### DISCUSSION THREAD

**Discussion by jaypeemayo** [OrchardCMS/OrchardCore#10741] [Q&A]
Example I have

Default Tenant - mywebsite.com
Tenant 1 - tenant1.website.com
Tenant 2 - anotherwebsite.com

Is this possible? and can you recommend a good guide of documentation for this?

**Comment by Skrypt** [comment]
This is what it is for. There is an issue though with the tenant setup url that needs to be fixed maybe that's why it seems to not work.

**Reply by Skrypt** [reply]
See https://github.com/OrchardCMS/OrchardCore/pull/10743

**Reply by jaypeemayo** [reply]
Thanks. hopefully it will be fixed soon @Skrypt
